### PR TITLE
Move audio decoders to xbmc organization

### PR DESCRIPTION
**Pull request by notspiff**
Also add 4 missing decoders.

### CODE REVIEW

File: audiodecoder.usf/platforms.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+all
```
Comment (Rechi) [inline audiodecoder.usf/platforms.txt:1]: This currently doesn't succeed for android and ios. Do you plan to fix it?

Comment (notspiff) [reply audiodecoder.usf/platforms.txt:1]: i don't think i can, i think there are some x86 assembly in there.

Comment (Rechi) [reply audiodecoder.usf/platforms.txt:1]: from a quick look it looks like it is caused by https://github.com/xbmc/audiodecoder.usf/blob/master/lib/lazyusf/CMakeLists.txt#L33-L39 which should be fixable.

Comment (notspiff) [reply audiodecoder.usf/platforms.txt:1]: okay, then let's try that.

Comment (Rechi) [reply audiodecoder.usf/platforms.txt:1]: fixed at xbmc/audiodecoder.usf/pull/2

**Pull request merged**